src/dropzone/dropzone.html

```diff
@@ -1,7 +1,6 @@
 <form submit-form
   action="{{ $ctrl.url }}"
   class="dropzone"
-  id="s3c-dropzone"
```

Comment:
This might as well be "fixup"ped into the commit that originally introduced this code.

There's no reason to have a commit that introduces a problem, and then a separate commit to fix that problem, as part of a single feature branch to merge.

Reply:
That makes sense to me, this commit has been merged into the commit where dropzone.html is introduced. Thanks for catching this.